sonar-db/src/main/resources/org/sonar/db/organization/OrganizationMapper.xml

```diff
@@ -99,6 +99,28 @@
     </where>
   </sql>
 
+  <select id="selectByPermission" parameterType="map" resultType="Organization">
+    select
+      distinct <include refid="selectColumns"/>
```

Comment:
Just for information, no need for `distinct` because SQL `union` returns a set, not a list.